alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultJobAccessor.java

```diff
@@ -50,6 +50,7 @@
 import com.synopsys.integration.blackduck.api.manual.enumeration.NotificationType;
 import com.synopsys.integration.datastructure.SetMap;
 
+// @Component
```

Comment:
Will this break during runtime? Or has all references to DefaultJobAccessor been removed/replaced?

Reply:
There is a different `@Component` that will be picked up for `JobAccessor`.